docs/sbom-tool-api-reference.md

```diff
@@ -217,7 +217,7 @@ var file = new SbomFile
 };
 ```
 
-The API looks for a relative path starting with a period `.`.  All path separators should include forward slashes `/` in compliance with the SPDX version 2.2 specification.
+The API looks for a relative path starting with a period `.`.  All path separators should include forward slashes `/` in conformance with the SPDX version 2.2 specification.
```

Comment:
This change can be reverted since it is unrelated to the SPDX 3.0 addition of compliance standard validation. This is something that already existed before my changes.